simulation/Map/map.py

```diff
@@ -0,0 +1,7 @@
+class Map:
+    def __init__(self, map_id, name, version, roads):
```

Comment:
make all variables private. e.g self.__id = map_id. make setters and getter.
The map folder should not be in simulation. Transfer it outside